app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInnerState.kt

```diff
@@ -18,18 +19,21 @@ import com.wire.android.util.toByteArray
 import java.io.IOException
 
 @Composable
-fun rememberMessageComposerInnerState(): MessageComposerInnerState {
+fun rememberMessageComposerInnerState(fullScreenHeight: Dp): MessageComposerInnerState {
     val defaultAttachmentInnerState = AttachmentInnerState(LocalContext.current)
 
     return remember {
-        MessageComposerInnerState(defaultAttachmentInnerState)
+        MessageComposerInnerState(fullScreenHeight,defaultAttachmentInnerState)
     }
 }
 
 class MessageComposerInnerState(
+    val fullScreenHeight: Dp,
```

Comment:
Is it used anywhere or will be? Should it really be full screen height or maybe without insets (status bar, navigation bar)?

Reply:
it is used inside MessageComposer, indeed it is without insets, will mention it in the name of it